.github/workflows/edr-npm-release.yml

```diff
@@ -10,6 +10,7 @@ permissions:
   push:
     branches:
       - main
+      - edr/ci/fix-missing-type-def-in-edr-package
```

Comment:
```suggestion
      - edr/main
```

Reply:
Fixed in [bf4e7f2](https://github.com/NomicFoundation/hardhat/pull/4512/commits/bf4e7f20c6691262160e41e78ab064072ed753c8)